pkg/app/launcher/cmd/launcher/binary_test.go

```diff
@@ -28,6 +28,10 @@ func TestCommand(t *testing.T) {
 	require.NotNil(t, cmd)
 
 	assert.True(t, cmd.IsRunning())
-	cmd.GracefulStop(time.Millisecond)
+
+	stopAfter := time.Millisecond
+	cmd.GracefulStop(stopAfter)
+	time.Sleep(stopAfter)
```

Comment:
We should avoid using `time.Sleep` in tests.
So how about adding `<- c.stoppedCh`  to before this line instead of fixing the test.
https://github.com/pipe-cd/pipe/blob/master/pkg/app/launcher/cmd/launcher/binary.go#L53

Reply:
That's better, thank you. Addressed by https://github.com/pipe-cd/pipe/pull/2758/commits/de5214bfb02e011c79cb0e0ebbb736b388e28839